src/shared/api/generated/CBioPortalAPI.ts

```diff
@@ -2357,6 +2436,11 @@ export default class CBioPortalAPI {
         }): Promise < Array < MolecularProfile >
         > {
             return this.fetchMolecularProfilesUsingPOSTWithHttpInfo(parameters).then(function(response: request.Response) {
+                response.body = _(response.body).each((d:MolecularProfile) => {
+                    if (d.sortOrder !== undefined) {
+                        d.sortOrder = d.sortOrder as any === "ASC"? SortOrder.ASC : SortOrder.DESC;
```

Comment:
did you hand write this?

Reply:
seems like you might want to move these and your changes above into another file in /api/ but not /generated/, like `TreatmentAPI` or something

Reply:
Treatment API was replaced by swagger CodeGen output.